src/walletSdk/Horizon/Stellar.ts

```diff
@@ -1,15 +1,76 @@
+import { Account as StellarAccount, Server, Transaction } from "stellar-sdk";
 import { Config } from "walletSdk";
 import { AccountService } from "./AccountService";
+import { TransactionBuilder } from "./transaction/TransactionBuilder";
+import { TransactionParams } from "../Types";
+import {
+  AccountDoesNotExistError,
+  TransactionSubmitFailedError,
+} from "../Exceptions";
 
 // Do not create this object directly, use the Wallet class.
 export class Stellar {
   private cfg: Config;
-  
+  server: Server;
+
   constructor(cfg: Config) {
     this.cfg = cfg;
+    this.server = cfg.stellar.server;
   }
 
   account(): AccountService {
     return new AccountService(this.cfg);
   }
+
+  async transaction({
```

Comment:
a single transaction method in this sdk ([kotlin has two](https://github.com/stellar/kotlin-wallet-sdk/blob/cf6ab208aa9ab2b60e0a691005ced6ba6f4219d7/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/horizon/Stellar.kt#L41-L72)), but the timebounds argument can be either a number or Timebounds object